datahub-web-react/src/app/entity/shared/GlossaryEntityContext.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+import { GenericEntityProperties } from './types';
+
+export interface GlossaryEntityContextType {
+    isInGlossaryContext: boolean;
+    entityData: GenericEntityProperties | null;
+    setEntityData: (entityData: GenericEntityProperties | null) => void;
+    updatedUrns: string[];
```

Comment:
These fields are less intuitive to understand than the others ... might be helpful to have comment

Reply:
+1